examples/remote_procedure/mnist/hydra_configs/conf/launch_with_local.yaml

```diff
@@ -22,3 +22,5 @@ mephisto:
     task_tags: "mnist,drawing,models,correction"
     # We expect to handle 25 people using the MNIST model at once
     max_num_concurrent_units: 25
+    post_install_script: ""
+    force_rebuild: false
```

Comment:
Given these have a default hydra value (introduced in `TaskRun`'s file) is it required that we include these defaults in all of these files? 

I imagine the tradeoff between discoverability and boilerplate is an important one to think about if not, but I'd lean towards these more landing in the camp of a heavy user or contributor.

Reply:
As most people will not do local package development, I would think it makes sense to remove them from the hydra config.

I just tested it, and it works without defining them in the hydra config(uses the TaskRun default value).